src/basic/terminal-util.c

```diff
@@ -1565,6 +1545,32 @@ int set_terminal_cursor_position(int fd, unsigned int row, unsigned int column)
         return 0;
 }
 
+int terminal_enable_line_wrapping_fd(int fd) {
```

Comment:
hmm, any chance you can rename this terminal_reset_ansi_seq() or so, to make clear this is about sending an ANSI Sequence?

also, could you maybe change the sequence to:

```c
"\033c"        /* reset to initial state */ 
"\033[!p"      /* soft terminal reset */
"\033]104\007" /* reset colors */
"\033[?7h"     /* enable line-wrapping */
```

This appears to be what "reset" actually does